pkg/ccl/logictestccl/testdata/logic_test/tenant_unsupported

```diff
@@ -53,9 +53,6 @@ SELECT * FROM crdb_internal.zones
 statement error operation is unsupported in multi-tenancy mode
 SHOW ZONE CONFIGURATION FOR TABLE kv
 
-statement error operation is unsupported in multi-tenancy mode
```

Comment:
Move this statement test to `pkg/ccl/logictestccl/testdata/logic_test/tenant` instead. I'd delete the `zone_config_tenants` test file for now. Also -- why does it show up as a binary file on github/locally? Try `$ file <path to zone_config_tenants>`. Is it because we're printing out the proto bytes directly? Seems bad for a logic test, printing out number of rows would be better. Better yet, test what's visible through `SHOW ZONE CONFIGURATIONS` instead. 

I think the only behavioural difference between secondary/host tenants will be around not being able to set/view the distinguished zone configs (liveness, meta -- see RFC), for those it makes sense to use a separate tenant-specific file. Could do it in this PR, or add a TODO(zcfgs-pod) somewhere.

```
statement ok
ALTER RANGE DEFAULT CONFIGURE ZONE USING num_replicas = 5;
```